src/main/java/com/microsoft/sqlserver/jdbc/ParameterMetaDataCache.java

```diff
@@ -31,17 +31,14 @@ class ParameterMetaDataCache {
      *        Array of parameters used
      * @param parameterNames
      *        Names of parameters used
-     * @param session
-     *        The current enclave session containing the cache
      * @param connection
      *        The SQLServer connection
      * @param stmt
      *        The SQLServer statement, whose returned metadata we're checking
      * @return true, if the metadata for the query can be retrieved
-     * 
      */
-    static boolean getQueryMetadata(Parameter[] params, ArrayList<String> parameterNames, CryptoCache cache,
-            SQLServerConnection connection, SQLServerStatement stmt) throws SQLServerException {
+    boolean getQueryMetadata(Parameter[] params, ArrayList<String> parameterNames, SQLServerConnection connection,

```

Comment:
I would keep existing static methods static. It may only be a small perf difference, but we definitely care about small perf changes in the driver.

Reply:
I ended up changing them back. Running tests with the methods as static and not, I couldn't see a difference, so I'll go with static since its cleaner.